integration/tests/epochs/suite.go

```diff
@@ -441,7 +465,7 @@ func (s *Suite) SetApprovedNodesScript(ctx context.Context, env templates.Enviro
 		SetProposalKey(s.client.SDKServiceAddress(), 0, s.client.Account().Keys[0].SequenceNumber).
 		SetPayer(s.client.SDKServiceAddress()).
 		AddAuthorizer(idTableAddress)
-
+	s.client.Account().Keys[0].SequenceNumber++
```

Comment:
Why do we need to increment the sequence number here, but not in `SubmitAdminRemoveNodeTx` (which also submits a transaction with the same account) ?

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/b8764d41aa39b48d9971e01f862b0c20d3804d86